bikeshed/shorthands.py

```diff
@@ -226,6 +226,16 @@ def elementReplacer(match):
     def varReplacer(match):
         return E.var(match.group(1))
 
+    strongRe = re.compile(r"([_\*])\1([\d\w\s]+)\1\1")
```

Comment:
No need to escape a `*` inside of a character class.

`\d\w\s` does not capture the full range of things you can put in there - for example, it excludes hyphens. You're gonna want a lookahead that just checks that the next two characters aren't `\1\1`.